mithril-aggregator/src/services/prover.rs

```diff
@@ -410,6 +412,50 @@ mod tests {
         transactions_set_proof[0].verify().unwrap();
     }
 
+    #[tokio::test]
+    async fn cant_compute_proof_for_not_yet_certified_transaction() {
+        let total_block_ranges = 5;
+        let total_transactions_per_block_range = 3;
```

Comment:
If the variable is kept, it should be renamed as follows
```suggestion
        let blocks_per_block_range = 3;
```